### ISSUE TITLE
Cannot create initial DB

### ISSUE THREAD

**Issue by audricschiltknecht** [charettes/django-mutant#44]
When starting a new project, the `manage.py makemigrations` command fails:

```
$ django-admin2 startproject mutantest
$ cd mutantest/
$ virtualenv venv
$ source venv/bin/activate
$ pip install django-mutant
$ vim mutantest/settings.py
Add 'mutant' to INSTALLED_APPS
$ pip list
Django (1.9.6)
django-mutant (0.2.1)
django-picklefield (0.3.2)
django-polymodels (1.4.4)
pip (8.1.2)
setuptools (21.2.1)
wheel (0.29.0)
$ ./manage.py makemigrations
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/core/management/commands/makemigrations.py", line 132, in handle
    migration_name=self.migration_name,
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/db/migrations/autodetector.py", line 46, in changes
    changes = self._detect_changes(convert_apps, graph)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/db/migrations/autodetector.py", line 132, in _detect_changes
    self.new_apps = self.to_state.apps
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/utils/functional.py", line 33, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/db/migrations/state.py", line 162, in apps
    return StateApps(self.real_apps, self.models)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/db/migrations/state.py", line 228, in __init__
    self.render_multiple(list(models.values()) + self.real_models)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/db/migrations/state.py", line 296, in render_multiple
    model.render(self)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/db/migrations/state.py", line 585, in render
    body,
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/db/models/base.py", line 292, in __new__
    new_class._meta.apps.register_model(new_class._meta.app_label, new_class)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/db/migrations/state.py", line 325, in register_model
    self.do_pending_operations(model)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/apps/registry.py", line 406, in do_pending_operations
    function(model)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/apps/registry.py", line 388, in function
    self.lazy_model_operation(next_function, *more_models)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/django/apps/registry.py", line 397, in lazy_model_operation
    function(model_class)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/polymodels/fields.py", line 113, in resolve_polymorphic_type
    field.validate_polymorphic_type(related_model)
  File "/tmp/mutantest/venv/lib/python2.7/site-packages/polymodels/fields.py", line 103, in validate_polymorphic_type
    "First parameter to `PolymorphicTypeField` must be "
AssertionError: First parameter to `PolymorphicTypeField` must be a subclass of `BasePolymorphicModel`

```

**Comment by charettes** [comment]
Moved to charettes/django-polymodels#29.

**Comment by charettes** [comment]
This should be fixed in 3.0a1 which is available on pypi.

**Comment by audricschiltknecht** [comment]
Indeed, thanks!
